mx.nodeprof/suite.py

```diff
@@ -6,7 +6,7 @@
     "suites" : [
       {
         "name" : "graal-nodejs",
-        "version" : "e256e1f1b27d29a92fe59fab5cc54fff60bae4a4",
+        "version" : "9a6cd9fc91dd8a516f18882d32321e293f118bdf",
```

Comment:
This is the release branch and it's missing instrumentation fixes by @eleinadani that have been merged since then. Did you mean to update the import to this GraalVM release version in particular, or just a newer version?

Reply:
I wanted to have a Github tag corresponding to the latest GraalVM release so that the results ran with GraalVM-rc11 will be consistent with the result ran with the built GraalVM. 
Of course, we can continue to update the Graal.js version with latest fixes in the master branch.